backfill_corrections/Makefile

```diff
@@ -1,8 +1,19 @@
 SHELL:=/bin/bash
 
-TODAY:=$(shell date -u +"%Y-%m-%d")
-CURR_TIME:=$(shell date -u +"%Hh%Mm%Ss")
-LOG_FILE:=$(TODAY)_$(CURR_TIME).log
+DOCKER_IMAGE=ghcr.io/cmu-delphi/covidcast-indicators-backfill_corrections
+DOCKER_TAG=latest
+# Change training options during `make` call via `make <command> OPTIONS="<options>"`
+OPTIONS=--train_models --make_predictions
+
+S3_URL=https://backfill_corrections.s3.us-east-2.amazonaws.com
+S3_BUCKET=s3://backfill_corrections
```

Comment:
All our other pipelines read S3 locations out of params.json; is there a compelling reason to hard-code these ones?

Reply:
Good point. I was modeling this after the `forecast-eval` Makefile. I'll take a look at other ways to set this up.